openrtb-core/src/main/protobuf/openrtb.proto

```diff
@@ -1126,6 +1126,12 @@ message BidRequest {
     // and MNC parts is required to remove parsing ambiguity.
     optional string mccmnc = 30;
 
+    // Structured user agent information defined by a UserAgent object
+    // If both ‘ua’ and ‘sua’ are present in the bid request, ‘sua’ should be considered
```

Comment:
Nitpick, let's please discuss using double quotes instead of single for consistency.